runtime/src/accounts_index.rs

```diff
@@ -1682,7 +1688,9 @@ impl<T: IndexValue> AccountsIndex<T> {
         let mut w_roots_tracker = self.roots_tracker.write().unwrap();
         // `AccountsDb::flush_accounts_cache()` relies on roots being added in order
         assert!(slot >= w_roots_tracker.roots.max_inclusive());
+        // 'slot' is a root, so it is both 'root' and 'original'
         w_roots_tracker.roots.insert(slot);
```

Comment:
nit: maybe add a function on roots_tracker for this and the following statement?